### PR TITLE
speed-test: add page

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/speed-test.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Test your internet connection speed and ping using speedtest.net from the CLI.
+> More information: <https://github.com/sindresorhus/speed-test/>.
+
+- Test your internet connection speed and ping:
+
+`speed-test`
+
+- Output the results as JSON:
+
+`speed-test --json -j`
```
Comment (bl-ue) [inline pages/common/speed-test.md:12]:
```suggestion
`speed-test --json`
```

File: pages/common/speed-test.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Output the results in megabytes per second (MBps):
+
+`speed-test --bytes -b`
```
Comment (bl-ue) [inline pages/common/speed-test.md:16]:
```suggestion
`speed-test --bytes`
```

File: pages/common/speed-test.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Output more detailed information:
+
+`speed-test --verbose -v`
```
Comment (bl-ue) [inline pages/common/speed-test.md:20]:
```suggestion
`speed-test --verbose`
```

File: pages/common/speed-test.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# speed-test
+
+> Test your internet connection speed and ping using speedtest.net from the CLI.
+> More information: <https://github.com/sindresorhus/speed-test/>.
```
Comment (bl-ue) [inline pages/common/speed-test.md:4]:
```suggestion
> More information: <https://github.com/sindresorhus/speed-test>.
```

File: pages/common/speed-test.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Test your internet connection speed and ping:
```
Comment (bl-ue) [inline pages/common/speed-test.md:6]:
```suggestion
- Test your internet connection speed and ping speed:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks like you left the short options on the examples as well as the long options ;)

**Review by owenvoke — APPROVED** [review]
LGTM after the changes @bl-ue suggested.

File: pages/common/speed-test.md (hunk 5 above)
Comment (Yutyo) [reply pages/common/speed-test.md:6]: What about making it "Test your internet connection and ping speed" instead, which is shorter?

Comment (bl-ue) [reply pages/common/speed-test.md:6]: Suggestion updated.

**Comment by Yutyo** [comment]
🦗

**Review by bl-ue — APPROVED** [review]

**Pull request merged**